### DISCUSSION TITLE
Can Spades be used for de novo assembly of RAD-seq Illumina reads?

### DISCUSSION THREAD

**Discussion by Anto007** [ablab/spades#855] [Q&A]
Hi, I would like to know if RAD-seq Illumina reads can be de novo assembled on Spades so as to produce contigs that match genes/genome of an organism of interest? There are tools such as Rainbow (https://academic.oup.com/bioinformatics/article/28/21/2732/233132) that seem to be specifically geared to address this but I'm just curious to know if Spades would work correctly if I input RAD-seq Illumina reads to it?

**Comment by asl** [comment]
Well, it may certainly work, however, it will certainly assemble only separate fragments.
